File: java/src/peachtree/aln/colourings/Jalview.java
package peachtree.aln.colourings;

import java.util.HashMap;
import java.util.Map;


public class Jalview extends Colouring {

	
	
	 
	
	
	 
	public Jalview() {
		
		
		 colours = new  HashMap<String, String>();
		 colours.put("A", "#64F73F");
		 colours.put("C", "#FFB340");
		 colours.put("G", "#EB413C");
		 colours.put("T", "#3C88EE");
		 colours.put("U", "#3C88EE");
		 //colours.put("N", "white");
		 
	}
	
	@Override
	public String getName() {
		return "Jalview";
	}

	@Override
	public boolean isNucleotide() {
		return true;
	}


	
	
	

	
	
	
	

}

File: cpp/peachtree/src/aln/colourings/JalView.cpp
/*
 * JalView.cpp
 *
 *  Created on: 8/07/2021
 *      Author: jdou557
 */

#include "JalView.h"

JalView::JalView() {
	colours["A"] = "#64F73F";
	colours["C"] = "#FFB340";
	colours["G"] = "#EB413C";
	colours["T"] = "#3C88EE";
	colours["U"] = "#3C88EE";
}




string JalView::getName(){
	return "JalView";
}


bool JalView::isNucleotide(){
	return true;
}
